### PR TITLE
Tweak monorepo section

### PR DESCRIPTION
**Pull request by sapegin**

### CODE REVIEW

File: manuscript/appendices/02-monorepos.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ npm packages can be managed in multiple ways. Perhaps the most common way is to
 
 ## Monorepos - What Are They
 
-A monorepo allows you to maintain multiple related packages within a single repository. Bigger projects, such as Babel or Jest, use this method for organizing their work. They use [Lerna](https://lernajs.io/), a tool designed to help maintaining monorepos. The tooling can figure out what packages to update based on changes and help with the versioning problem.
+A monorepo allows you to maintain multiple related packages within a single repository. Bigger projects, such as Babel or Jest, use this method for organizing their work. They use [Lerna](https://lernajs.io/), a tool designed to help maintaining monorepos. It can figure out what packages to update based on changes and help with the versioning problem.
```
Comment (bebraw) [inline manuscript/appendices/02-monorepos.md]: I would go with "Lerna" over "It" for clarity. I think I used "The tooling" as it's more generic but now that I see it, it was too generic.

File: manuscript/appendices/02-monorepos.md [hunk 2]
```diff
@@ -14,59 +14,36 @@ To give you an idea of a monorepo, consider the following structure from Reactab
...
 
 Separating each package to a repository of its own makes them snowflakes in sense that each repository easily becomes different and it's hard to keep them in sync if infrastructure evolves somehow. To solve the problem, [webpack-defaults](https://www.npmjs.com/package/webpack-defaults) was developed.
 
-The idea was to push project configuration (GitHub templates, linting, testing setup, i.e.) to a single package that could be consumed across webpack-contrib repositories. When you install *webpack-defaults* to your project, it writes a `webpack-defaults` npm script entry you can run. After you perform `npm run webpack-defaults`, it pulls project defaults from the package and migrates the project as well it can to follow the standard.
+The idea was to push project configuration (GitHub templates, linting, testing setup, etc.) to a single package that could be consumed across webpack-contrib repositories. When you install *webpack-defaults* to your project, it writes an npm script that pulls project defaults from the package and migrates the project as well it can to follow the standard.
```
Comment (bebraw) [inline manuscript/appendices/02-monorepos.md]: Note that I use `i.e.` and `e.g.` on purpose. It's a stylistic choice and we should lint possible `etc.` away (replace with `i.e.` literally).

Comment (sapegin) [inline manuscript/appendices/02-monorepos.md]: But there are all not the same and all are equally ugly if that was the reason not to use etc. ;-) 

So in this case it’s either “e.g., GitHub templates, linting, testing setup” or “GitHub templates, linting, testing setup, etc.). In any case we need a style guide to document such decisions.

Comment (bebraw) [inline manuscript/appendices/02-monorepos.md]: I would ban `etc.` in any case. I agree going with `i.e.` and `e.g.` feels weird at first but you get used to them and it seems to be good style. `etc.` would work for a blog post but not for a book.

These can be pushed to textlint.

Comment (sapegin) [inline manuscript/appendices/02-monorepos.md]: I disagree that i.e./e.g. is stylistically different from etc. but agree to prefer one style ;-)

Comment (bebraw) [inline manuscript/appendices/02-monorepos.md]: Older material is on i.e./e.g. . Even if it's not perfect, let's lint for it and go with this.

There's one thing about etc. that's good to keep in mind. It usually tells the writer got lazy. Often you can find a better way out and can rewrite the idea without it. Just realized this is two separate issues in one.

Comment (sapegin) [inline manuscript/appendices/02-monorepos.md]: Yup, that’s why I say there’s no difference ;-)

Comment (bebraw) [inline manuscript/appendices/02-monorepos.md]: Ok, fair enough. 👍

**Comment by sapegin** [comment]
Please look again ;-)

**Pull request merged**